cas-airflow/values.yaml

```diff
@@ -1,33 +1,90 @@
 airflow:
+  web:
+    initialDelaySeconds: 60
+  dags:
+    git:
+      url: https://github.com/bcgov/cas-airflow-dags.git
+      ref: master
+    initContainer:
+      enabled: true
+      installRequirements: false
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.033'
+    executor: Kubernetes
+    config:
+      AIRFLOW__CORE__REMOTE_LOGGING: True
+      AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER: gs://wksv3k-tools-app1
+      AIRFLOW__CORE__REMOTE_LOG_CONN_ID: gcs_logs
+      # AIRFLOW__KUBERNETES__GIT_REPO: https://github.com/bcgov/cas-airflow-dags.git
+      # AIRFLOW__KUBERNETES__GIT_BRANCH: master
+      # AIRFLOW__KUBERNETES__GIT_DAGS_FOLDER_MOUNT_POINT: /usr/local/airflow/dags
+      AIRFLOW__KUBERNETES__KUBE_CLIENT_REQUEST_ARGS: '{\"_request_timeout\":[300,300]}'
```

Comment:
should already be covered by `kube_client_request_args` in custom `airflow.cfg` now injected into docker image

Reply:
Ah, I was debugging a different problem and decided to add this back in to see if it made a difference, it could probably be removed